R/expr__datetime.R

```diff
@@ -89,47 +89,40 @@ ExprDT_round = function(every, offset = NULL) {
     unwrap("in $dt$round()")
 }
 
-# ExprDT_combine = function(self, tm: time | pli.RPolarsExpr, tu: TimeUnit = "us") -> pli.RPolarsExpr:
-
-
-#' Combine Data and Time
-#' @description  Create a naive Datetime from an existing Date/Datetime expression and a Time.
-#' Each date/datetime in the first half of the interval
-#' is mapped to the start of its bucket.
-#' Each date/datetime in the second half of the interval
-#' is mapped to the end of its bucket.
+#' Combine Date and Time
 #'
+#' Create a naive Datetime from an existing Date/Datetime expression and a Time.
+#' Each date/datetime in the first half of the interval is mapped to the start
+#' of its bucket. Each date/datetime in the second half of the interval is
+#' mapped to the end of its bucket.
 #'
-#' @param tm Expr or numeric or PTime, the number of epoch since or before(if negative) the Date
-#' or tm is an Expr e.g. a column of DataType 'Time' or something into an Expr.
-#' @param tu time unit of epochs, default is "us", if tm is a PTime, then tz passed via PTime.
+#' @param time The number of epoch since or before (if negative) the Date. Can be
+#' an Expr or a PTime.
+#' @inheritParams DataType_Datetime
 #'
+#' @inherit ExprDT_truncate details return
```

Comment:
I don't think it is possible to inherit details from `ExprDT_truncate` because the argument names are different.

Reply:
Good catch, I think I got confused actually because none of the args uses the string duration language. I removed it.